README.rst

```diff
@@ -0,0 +1,99 @@
+Before the start, make sure that you have an account on `PyPI`_
+
+
+Quickstart
+----------
+ 
+1. Install Cookiecutter
+
+Create and activate an environment for the package project.
+
+.. code-block:: bash
+
+    conda create -n <repo_name> python=<python_version>
+
+Activate your environment.
+
+.. code-block:: bash
+
+    activate <repo_name>          #Windows
+    source activate <repo_name>   #Linux
+    
+Install cookiecutter.
+
+.. code-block:: bash
+
+    conda install cookiecutter
+
+
+2. Create the package
+
+Use cookiecutter, informing the address of `cookiecutter-esss-pypackage`
+
+.. code-block:: bash
+
+    cookiecutter cookiecutter https://github.com/ESSS/cookiecutter-esss-pypackage
+
+You'll be asked to enter:
+
+- project_name
+- package_name
+- package_description
+- pypi_username
+
+
+3. Create a GitHub Repo
+
+Go to GitHub and create a new repository, where the name of the repository must be the same that ``[project_slug]`` from your answers to running cookiecutter. 
```

Comment:
Also add the the user should choose No to the GitHub options of adding a license and .gitignore.

Reply:
The following content was added:

When creating the repository, make sure to:
- Keep the `Initialize this repository with a README` option __unmarked__ 
- Keep the option `Add .gitignore` as `None`
- Keep the option `Add a License` as `None`